src/Tile/TileGroup.svelte

```diff
@@ -17,7 +17,7 @@
   const dispatch = createEventDispatcher();
   const selectedValue = writable(selected);
 
-  setContext("TileGroup", {
+  setContext("RadioTileGroup", {
```

Comment:
Refrain from making changes to TileGroup if there is already a dedicated RadioTileGroup component

Reply:
I did update TileGroup to use this key so its backwards compatible? (just aligning naming convention for when it depreciates)